sqlglot/parser.py

```diff
@@ -92,6 +92,9 @@ class Parser(metaclass=_Parser):
             length=exp.Literal.number(10),
         ),
         "VAR_MAP": parse_var_map,
+        "TRUNC": lambda args: exp.DateTrunc(
```

Comment:
delete this, and add a mapping in doris, 

"TRUNC": exp.DateTrunc.from_args_list

Reply:
I want to convert from other dialects to doris, only add the mapping in doris, it will not work for other dialects

Reply:
not every dialect supports trunc

Reply:
OK, the oracle dialect supports it, then I will write it in oracle